2025-05-12 21:14:53.948306+02:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO Usuario (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:14:53.953306+02:00 jdbc[5]: null
org.h2.message.DbException: La conexi贸n no fue cerrada por la aplicaci贸n y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-224]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:218)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:135)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at org.example.data.db.Database.getConnection(Database.kt:13)
	at org.example.data.dao.UsuariosDAOH2.insert(UsuariosDAOH2.kt:43)
	at org.example.service.UsuarioService.insert(UsuarioService.kt:14)
	at org.example.MainKt.main(Main.kt:19)
	at org.example.MainKt.main(Main.kt)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La conexi贸n no fue cerrada por la aplicaci贸n y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 13 more
2025-05-12 21:14:53.955307+02:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO Usuario (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:14:53.983312+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO Usuario (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:14:54.012319+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
SELECT * FROM Usuario [42104-224]
2025-05-12 21:16:18.331437+02:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO Usuario (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:16:18.335439+02:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO Usuario (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:16:18.336441+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO Usuario (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:16:18.340439+02:00 jdbc[7]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
SELECT * FROM Usuario [42104-224]
2025-05-12 21:17:29.168452+02:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO USUARIO (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:17:29.172452+02:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO USUARIO (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:17:29.173453+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO USUARIO (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:17:29.176453+02:00 jdbc[7]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
SELECT * FROM Usuario [42104-224]
2025-05-12 21:17:50.780511+02:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO USUARIO (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:17:50.786514+02:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO USUARIO (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:17:50.787514+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
INSERT INTO USUARIO (nombre, email) VALUES (?,?) [42104-224]
2025-05-12 21:17:50.792515+02:00 jdbc[7]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
SELECT * FROM Usuario [42104-224]
2025-05-13 14:00:45.833406Z jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1624)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-05-13 14:00:46.092717Z jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found (this database is empty); SQL statement:
SELECT t.*
FROM PUBLIC.USUARIO t
FETCH FIRST 501 ROWS ONLY [42104-232]
2025-05-13 14:04:14.526082Z jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1624)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
